querydsl-apt/pom.xml

```diff
@@ -79,6 +79,15 @@
       <artifactId>morphia</artifactId>
       <version>${morphia.version}</version>
       <scope>provided</scope>
+    </dependency>
+    <dependency>

```

Comment:
Consider using a different means of logging, the querydsl-apt transitive dependencies should be quite minimal.